x-pack/plugins/security_solution/common/endpoint/types.ts

```diff
@@ -350,7 +351,23 @@ export interface AlertEvent {
 }
 
 /**
- * The status of the host
+ * The status of the Endpoint Agent as reported by the Agent or the
+ * Security Solution app using events from Fleet.
+ */
+export enum EndpointStatus {
+  /**
+   * Agent is enrolled with Fleet
+   */
+  ENROLLED = 'enrolled',
```

Comment:
can we have the enum keys in lower case and match the value?

When (if) try to use this on the front-end it makes it easier to work with when we are working with the value.